### DISCUSSION TITLE
v6.3.7 broken in bun

### DISCUSSION THREAD

**Discussion by wenerme** [mikro-orm/mikro-orm#5986] [Q&A]
### Describe the bug

run in bun 1.1.26

> Entity X has wrong unique definition: 'cid' does not exist. You need to use property name, not column name.

but works under node v20.16.0

mikroorm 6.3.6 work fine under bun with same version.

### Reproduction

Just found the problem, due to affect the exists project, I'm still not sure why, I'll try to make a minimal repro.

### What driver are you using?

@mikro-orm/postgresql

### MikroORM version

6.3.7

### Node.js version

20.16.0

### Operating system

*

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Discord](https://discord.gg/w8bjxFHS7X).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
> Just found the problem, due to affect the exists project, I'm still not sure why, I'll try to make a minimal repro.

Please don't open issues without any reproduction.
